developer-docs-site/docs/move/move-on-aptos/cryptography.md

```diff
@@ -0,0 +1,211 @@
+---
+title: "Cryptography"
+slug: "cryptography"
+---
+
+# Cryptography in Move
+
+The Aptos adapter for Move has introduced many useful cryptographic primitives:
+
+1. Cryptographic hash functions (e.g., SHA2-256, SHA3-256, Keccak256, Blake2b-256)
+2. Digital signature verification (e.g., Ed25519, ECDSA, BLS)
+3. Elliptic curve arithmetic (e.g., Ristretto255, BLS12-381)
+4. Groth16 zero-knowledge proof (ZKP) verification
+5. Bulletproofs zero-knowledge range proof (ZKRP) verification
```

Comment:
```suggestion
## Cryptographic Primitives

Move, through the Aptos adapter, encompasses several fundamental cryptographic tools:

1. Cryptographic Hash Functions – These are algorithms designed to produce a fixed-size output (hash) from variable-size input data. Supported functions include SHA2-256, SHA3-256, Keccak256, and Blake2b-256.
2. Digital Signature Verification – This process ensures data integrity, authentication, and non-repudiation. Supported signature schemes include Ed25519, ECDSA, and BLS.
3. Elliptic Curve Arithmetic – Elliptic curves are used as the foundation for various cryptographic operations, such as generating keys, performing digital signatures, and verifying signatures. Supported curves include Ristretto255 and BLS12-381.
4. Zero-Knowledge Proofs (ZKP) and Zero-Knowledge Range Proofs (ZKRP) – These cryptographic techniques allow a party to prove a statement's truth without revealing the statement itself. Supported methods include Groth16 ZKP verification and Bulletproofs ZKRP verification.
```

Reply:
Created a heading for this section to separate it from the intro.
Added some context to each bullet and combined ZKP and ZKRP into one bullet.

@alinush please modify if you disagree with my descriptions.

Reply:
nice